### DISCUSSION TITLE
Issue with _id, and PCAP pulling data with Gigamon HC1 TAP network traffic.

### DISCUSSION THREAD

**Discussion by IpUnNum** [Security-Onion-Solutions/securityonion#5647] [Unsupported Versions]
Security Onion version: 2.3.70 
Install source: Security Onion ISO
Install type: 1 Manager Search, 4 Forward Nodes
Does so-status show all services running? Yes
Do you get any failures when you run salt-call state.highstate from the node? No 
Does the grid status show any failures?  No
Explain your issue. 

Issues with Kibana, and PCAP parsing data for proper filtering. In Kabana if I click any _id to look at traffic, the results show “no data available”, and in PCAP if I specify any any information beside the sensor, and time frame such as Ports, or Ips It also comes back as “no data available” . The work around I am doing for now is if I see something suspicious regarding connections that I use PCAP with just a time frame of the traffic, and open with Wireshark to do the pcap analysis filtering. 

I have two networks that are nearly identical that I work on One is uses a SPAN sessions and does not have this issue with _id, and PCAP, the other use  Gigamon HC1s to pull the traffic, I checked  Gigamon HC1's traffic with Wireshark, and it all looks like normal traffic that is being pushed out to the forward nodes.  I have completely rebuilt the distributed deployment 3 times that utilize the  Gigamon HC1s and the issue happens each time.  But not at all with the SPAN method.

Has anyone else faced a similar issue that they managed to fix? or thoughts I should look into to investigate what is causing this issue?

When I was using 16.04 with a similar set up this was not an issue.

**Comment by dougburks** [comment]
Is the Gigamon doing any encapsulation of the network traffic?
